index.php

```diff
@@ -9,6 +9,10 @@
 
 $assetService = AssetService::getInstance();
 
+if ($config['ui']['local_timezone']) {
```

Comment:
Enough to put inside lib/boot.php ? Else we also need it on chroma etc

Reply:
i have the problem, when in php.ini is set date.timezone then its override the date_default_timezone_set. if I'm not mistaken